central/graphql/resolvers/image_vulnerabilities.go

```diff
@@ -91,8 +159,69 @@ func (resolver *Resolver) ImageVulnerabilityCounter(ctx context.Context, args Ra
 		query := withImageCveTypeFiltering(args.String())
 		return resolver.vulnCounterV2(ctx, RawQuery{Query: &query})
 	}
-	// TODO add postgres support
-	return nil, errors.New("Resolver ImageVulnCounter does not support postgres yet")
+
+	// check permissions
+	if err := readImages(ctx); err != nil {
+		return nil, err
+	}
+
+	// cast query
+	query, err := args.AsV1QueryOrEmpty()
+	if err != nil {
+		return nil, err
+	}
+
+	// get loader
+	loader, err := loaders.GetImageCVELoader(ctx)
+	if err != nil {
+		return nil, err
+	}
+	query = tryUnsuppressedQuery(query)
+
+	// get fixable vulns
+	fixableQuery := search.ConjunctionQuery(query, search.NewQueryBuilder().AddBools(search.Fixable, true).ProtoQuery())
```

Comment:
Make sure `query` does not have `search.Fixable`. If the query has `Fixable=false` then the resolver will return an unexpected response. 
Ideally, we should throw an error to the user. Someone querying this field and passing a fixable query is misuse. But for now log an error because I feel UI could be passing incorrect queries and we do not want to fail.

Reply:
How do you tell if there's an already included scoping on fixable? I feel like the easiest way to do this would be to just remove filtering on fixable for this resolver since we expressly pass back fixable and unfixable counts